src/frequenz/sdk/actor/power_distributing/_component_managers/_ev_charger_manager/_ev_charger_manager.py

```diff
@@ -350,6 +351,12 @@ async def _set_api_power(
                         component_id,
                         err,
                     )
+                case e if e is not None:
```

Comment:
This will catch `BaseException` too, is it intended? Maybe it doesn't make sense to log if a `SystemExit` or `KeyboardInterrupt` are raised (for example).
```suggestion
                case Exception():
```

Reply:
updated.